content/enterprise_influxdb/v1.9/guides/fine-grained-authorization.md

```diff
@@ -48,8 +48,18 @@ In a typical cluster configuration, the HTTP ports for data nodes
 You may need to work with your network administrator to gain access to the meta node HTTP ports.
     {{% /note %}}
 
-4. _(Optional)_ [Create roles](#manage-roles).
-   Roles let you grant permissions to groups of users assigned to each role.
+4. [Create roles](#manage-roles). Roles let you grant permissions to groups of users assigned to each role.
+   As administrator, create a user with a standard grant, as shown in the example below, and then restrict by role or via grants. 
+   You **must** set up coarse read/write permissions **FIRST**, then restrict to a fine grain.
```

Comment:
- Let's add steps to indicate required order:

4.
   ..a.   
   ..b.

- Let's use replace "set up coarse" to "grant all" (more explicit)

Reply:
Re:
create a user with a standard grant, as shown in the example below, and then restrict by role or via grants.

The example below grants users `east` and `west` all permissions (insert reference link to all permissions) on the `datacenters` database. 

And then, let's add a fine grain permission to one of the users for clarity (in example and text).